Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:lts AS build
+RUN --mount=type=secret,id=public_github_token,env=PUBLIC_GITHUB_TOKEN
```

Hadolint (2.14.0)

[error] missing whitespace

(DL1000)